src/Sentry.Extensions.Logging/SentryLoggerFormatter.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace Sentry.Extensions.Logging;
+
+internal class SentryLoggerFormatter
+{
+    public static readonly SentryLoggerFormatter Instance = new();
+
+    private SentryLoggerFormatter() {}
+
+    public string Invoke<TState>(TState state)
+    {
+        var dictionary = (state as IEnumerable<KeyValuePair<string, object>>)?.ToDictionary(i => i.Key, i => i.Value) ?? new Dictionary<string, object>();
```

Comment:
These should be `<string, object>`

Reply:
I don't believe we can guarantee that `TState` will be castable to `IEnumerable<KeyValuePair<string, object?>>` in all cases.  The contract for `TState` doesn't constrain it.  In theory, any caller to `ILogger.Log` could pass a different type of data, as long as the formatter they supply can handle it.